### PR TITLE
docs: JK-10248: add two new partition size metrics

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers**

- SME review already completed offline. See this [Google doc](https://docs.google.com/document/d/1ebxyWxPQaVRNKH9BnZHEe5FKe7JloVNr0kd7rN72DJc/edit?usp=sharing).
- An HTML preview is available for review [here](http://file.emea.redhat.com/~jbyrne/JK-10248/Monitoring%20metrics%20in%20OpenShift%20Streams%20for%20Apache%20Kafka.html).

### CODE REVIEW

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 1]
```diff
@@ -127,6 +127,10 @@ The Kafka instance type determines the maximum connection creation rate.
 `kafka_topic:kafka_topic_partitions:sum`:: Number of partitions across all topics in the cluster. This metric does not include partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`.
 +
 The Kafka instance type determines the maximum number of partitions.
+
+`kas_broker_partition_log_size_bytes_top50`:: Sizes, in bytes, of the fifty largest topic partitions on each broker in the cluster. The total amount of storage being used by all topic partitions on a broker is shown by the `kafka_broker_quota_totalstorageusedbytes` topic metric. The total usage for a broker must stay below the `kafka_broker_quota_softlimitbytes` value to avoid throttling of producers.
```
Comment (AedinC) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:131]: Do you mean the `kafka_broker_quota_totalstorageusedbytes`_broker_ metric here?

Comment (AedinC) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:131]: Suggest putting the last sentence about avoiding throttling into a note for added emphasis.

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 2]
```diff
@@ -127,6 +127,10 @@ The Kafka instance type determines the maximum connection creation rate.
 `kafka_topic:kafka_topic_partitions:sum`:: Number of partitions across all topics in the cluster. This metric does not include partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`.
 +
 The Kafka instance type determines the maximum number of partitions.
+
+`kas_broker_partition_log_size_bytes_top50`:: Sizes, in bytes, of the fifty largest topic partitions on each broker in the cluster. The total amount of storage being used by all topic partitions on a broker is shown by the `kafka_broker_quota_totalstorageusedbytes` topic metric. The total usage for a broker must stay below the `kafka_broker_quota_softlimitbytes` value to avoid throttling of producers. 
+
+`kas_topic_partition_log_size_bytes`:: Size, in bytes, of each topic partition on each broker in the cluster. The total amount of storage being used by all topic partitions on a broker is shown by the `kafka_broker_quota_totalstorageusedbytes` broker metric. The total usage for a broker must stay below the `kafka_broker_quota_softlimitbytes` value to avoid throttling of producers.
```
Comment (AedinC) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:133]: Same as previous comment.

**Review by AedinC — APPROVED** [review]
Looks good John - thanks for making these updates. Just a few minor comments.

File: docs/kafka/metrics-monitoring-kafka/README.adoc (hunk 1 above)
Comment (jbyrne-redhat) [reply docs/kafka/metrics-monitoring-kafka/README.adoc:131]: I absolutely did mean _broker_ metric. Thanks so much for catching that!

Comment (jbyrne-redhat) [reply docs/kafka/metrics-monitoring-kafka/README.adoc]: Yeah, I did consider that. But it's stated here twice with these two metrics. And then it's repeated again directly underneath, in the **Broker metrics** section. I think that's sufficient.

**Pull request merged**